flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/source/config/MySqlSourceOptions.java

```diff
@@ -127,6 +127,12 @@ public class MySqlSourceOptions {
                     .defaultValue(20)
                     .withDescription("The connection pool size.");
 
+    public static final ConfigOption<Integer> OUTPUT_RATE_LIMIT =
+            ConfigOptions.key("out.rate-limit")
+                    .intType()
+                    .defaultValue(0)
```

Comment:
```suggestion
                    .noDefaultValue()
```
I think this config option with no default value will be better.

Reply:
I tried to modify it, but I found that there are some 'nullpointerException', so I think the default value is 0